docker/gaffer-kerberos/README.md

```diff
@@ -0,0 +1,46 @@
+Gaffer with Kerberos
+======
+This folder contains images which build on those in the parent directory to use Kerberos for authentication.
+There's also an extra Dockerfile and image to implement a Kerberos KDC.
+
+### Purpose
+
+These images are for testing purposes only.
+They are only to be used for confirming that Gaffer's support for authenticating with Accumulo using Kerberos is working correctly.
+Configuration has been simplified as much as possible and is not indicative of how a Kerberos environment should be setup. 
+
+### Kerberos Debugging
+
+To see more Kerberos information in the container logs, set `DEBUG`to `1` in the docker compose `.env` file.
```

Comment:
Would this work with `true` and `false` instead?

Reply:
It would require changes elsewhere but it could use this. I'm not sure that true and false is better as it will just be a bash script looking at a string match.

Reply:
Yeah it's still not ideal in bash but I think maybe it is more clear than 1 and 0